projects/laji/test/+theme/genetic-resource/genetic-resource.po.ts

```diff
@@ -0,0 +1,26 @@
+import { Page } from '@playwright/test';
+
+export class GeneticResourcePage {
+  public $spinners = this.page.locator('laji-observation-result').locator('.spinner');
+
+  public hasNonGQLApiErrors = false;
+
+  constructor(
+    private page: Page
+  ) {
+    this.page.on('response', resp => {
```

Comment:
Maybe this could be extracted as a helper function instead? It seems a bit strange that the user of GeneticResourcePage must be aware of when the value of `hasNonGQLApiErrors` changes. So basically they have to read the implementation to use the class.

Reply:
could be something like `export const responseHasNonGQLApiErrors = (response): boolean => ...`

and then this.page.on... would be called in genetic-resource.spec.ts

Reply:
Actually it's probably fine since the `hasNonGQLApiErrors` is always up to date 🤔 However, resp should be typed with `import { Response } from '@playwright/test'`